hyundai_kia_generic.dbc

```diff
@@ -1451,6 +1450,17 @@ BO_ 909 FCA11: 8 FCA
  SG_ Supplemental_Counter : 35|4@1+ (1,0) [0|15] "" XXX
  SG_ PAINT1_Status : 16|2@1+ (1,0) [0|1] "" XXX
 
+BO_ 1156 HDA11_MFC: 8 XXX
+ SG_ Counter : 5|4@0+ (1,0) [0|15] "" XXX
+ SG_ NEW_SIGNAL_1 : 1|2@0+ (1,0) [0|255] "" XXX
+ SG_ NEW_SIGNAL_6 : 16|2@1+ (1,0) [0|3] "" XXX
+ SG_ NEW_SIGNAL_3 : 49|2@0+ (1,0) [0|1] "" XXX
+ SG_ NEW_SIGNAL_7 : 18|14@1+ (1,0) [0|63] "" XXX
+ SG_ NEW_SIGNAL_8 : 15|8@0+ (1,0) [0|255] "" XXX
+ SG_ NEW_SIGNAL_2 : 50|14@1- (1,-4095) [0|16383] "" XXX
+ SG_ NEW_SIGNAL_4 : 34|14@1+ (1,0) [0|16383] "" XXX
+ SG_ NEW_SIGNAL_5 : 33|2@0+ (1,0) [0|1] "" XXX
+ SG_ NEW_SIGNAL_9 : 7|2@0+ (1,0) [0|3] "" XXX
```

Comment:
There's no need to define these signals if we don't know what they are.

Reply:
It allows you to know for sure what the message boundary is. These are actually lane line outputs and the vehicle position relative to them. I have yet to figure out what lane each message follows.

Reply:
That's fine, just give them a name that indicates what we think they are and maybe add a comment line